text/0000-tiered-browser-support.md

```diff
@@ -0,0 +1,277 @@
+not currently doing any cross-browser testing; there are some JavaScript tests
+of the UI that run in [PhantomJS][], a headless WebKit. Since we're not
+testing, we don't actually know what our current web support even is, except
+for when we get bug reports from users.
+
+[PhantomJS]: http://phantomjs.org/
+
+In order to fully test on all browsers to be sure of our support, we would
+either need to have all the devices, operating systems, browsers, and versions
+available and people with the time and inclination to do manual testing on all
+of these, or we would need to be running automated tests on something like
+[BrowserStack][]. BrowserStack does appear to have a free plan for open source
+projects, but it's unclear how many parallel tests the open source plan would
+give us, and we'd at least be spending time waiting for test results on the
+various stacks. [BrowserStack also doesn't support every platform][bs-support],
+Linux on the desktop being a notable section of our userbase missing from their
+platforms.
+
+[BrowserStack]: https://www.browserstack.com/pricing
+[bs-support]: https://www.browserstack.com/support
+
+# Detailed design
+[design]: #detailed-design
+
+## Rust web content
+
+Officially produced web content includes:
+
+- rust-lang.org
+- blog.rust-lang.org
+- play.rust-lang.org
+- crates.io
+- Rustdoc output
+- thanks.rust-lang.org
+
+Explicitly not included:
+
+- Content for people working on Rust itself, such as:
+  - [The Rust Forge][]
+  - [rusty-dash][]
+
+[The Rust Forge]: https://forge.rust-lang.org/
+[rusty-dash]: https://rusty-dash.com/
+
+Things that are not really under our control but are used for official or
+almost-official Rust web content:
+
+- GitHub
+- docs.rs
+- Discourse (used for [urlo][] and [irlo][])
+- [mdBook][] output (used for the books and other documentation)
+
+[urlo]: https://users.rust-lang.org/
+[irlo]: https://internals.rust-lang.org/
+[mdBook]: https://github.com/azerupi/mdBook/
+
+## Proposed browser support tiers
+
+Based on [actual usage metrics][] and with a goal of spending our time in an
+effective way, the browser support tiers would be defined as:
+
+[actual usage metrics]: #google-analytics-browser-usage-stats
+
+Browsers are listed in [browserslist][] format.
+
+[browserslist]: https://github.com/ai/browserslist
+
+### Supported browsers
+
+Goal: Ensure functionality of our web content for 80% of users.
+
+Browsers:
+
+```
+last 2 Chrome versions
+last 1 Firefox version
+Firefox ESR
+last 1 Safari version
+last 1 iOS version
+last 1 Edge version
+last 1 UCAndroid version
+```
+
+[On browserl.ist](http://browserl.ist/?q=last+2+Chrome+versions%2C+last+1+Firefox+version%2C+Firefox+ESR%2C+last+1+Safari+version%2C+last+1+iOS+version%2C+last+1+Edge+version%2C+last+1+UCAndroid+version)
+
+Support:
+
+- We add automated testing for each of these as much as possible (and work on
+  adding automated testing to those web contents that aren't currently tested,
+  such as rustdoc output).
```

Comment:
Ok, I see your point this time. Then it might be worth it to specify it. So that we all agree on what testing the output is. :)